.github/workflows/ci.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         continue-on-error: true
 
       - name: prettier
-        run: yarn prettier:check '!ADOPTERS.md'
+        run: yarn prettier:check
```

Comment:
This is actually intended behaviour. New users already have a hard time signing the DCO and we don't want to make it any harder by ensuring the correct format. I'd say can add it to the prettierignore file instead and then manually remove it to format once in a while.

Reply:
makes sense. I wrote about these two options at Discord, but then thought I just make a PR with one of them.

I will rebase, rerun the check and add it to the prettierignore file.

Reply:
I skipped the rebase on master due to the (additional) prettier issues contained there. Will provide a separate PR for it.

Reply:
PR #10676

Reply:
I abandoned the other PR. Seems I missed a yarn install before. Also, seems like #10588 was fixing some of these things already

Reply:
Did the rebase and removed unnecessary changes.